packages/webapp/pages/account/others.tsx

```diff
@@ -1,55 +0,0 @@
-import { Switch } from '@dailydotdev/shared/src/components/fields/Switch';
-import AuthContext from '@dailydotdev/shared/src/contexts/AuthContext';
-import useProfileForm from '@dailydotdev/shared/src/hooks/useProfileForm';
-import React, { ReactElement, useContext, useState } from 'react';
-import { TimezoneDropdown } from '@dailydotdev/shared/src/components/widgets/TimezoneDropdown';
-import { getUserInitialTimezone } from '@dailydotdev/shared/src/lib/timezones';
-import { getAccountLayout } from '../../components/layouts/AccountLayout';
-import { AccountPageContainer } from '../../components/layouts/AccountLayout/AccountPageContainer';
-import AccountContentSection from '../../components/layouts/AccountLayout/AccountContentSection';
-
-const AccountOthersPage = (): ReactElement => {
-  const { user } = useContext(AuthContext);
-  const [userTimeZone, setUserTimeZone] = useState<string>(
-    getUserInitialTimezone({
-      userTimezone: user?.timezone,
-      update: true,
-    }),
-  );
-  const { updateUserProfile } = useProfileForm();
-
-  return (
-    <AccountPageContainer title="Other settings">
-      <AccountContentSection
-        className={{ heading: 'mt-0' }}
-        title="Timezone"
-        description="Used to calculate your weekly goal cycle and other time-based
-        activities."
-      >
-        <TimezoneDropdown
-          userTimeZone={userTimeZone}
-          setUserTimeZone={setUserTimeZone}
-        />
-      </AccountContentSection>
-      <AccountContentSection title="Newsletter">
```

Comment:
And this is totally removed we don't place it somewhere else?

Reply:
Forgot to mention, but yeah totally not used anymore:
Community newsletter setting - delete (no longer used)